webapp/lib/controller.dart

```diff
@@ -120,24 +118,24 @@ void loadProjectSelectorView() {
 void loadDashboardView() {
   view.navView.projectTitle = project?.projectName;
   view.navView.projectOrganizations = [''];
-  var dashboardView = new view.DashboardView();
+  var dashboardView = new view.DashboardView(model.conversationData);
   dashboardView.activePackages.addAll(
     [
-      new view.ActivePackagesViewPartial('Urgent conversations', '#/conversations', '#/escalates-configuration'),
-      new view.ActivePackagesViewPartial('Open conversations', '#/conversations', '#'),
-      new view.ActivePackagesViewPartial('Batch replies (Week 12)', '', '#/batch-replies-configuration'),
+      new view.ActivePackagesViewPartial('Urgent conversations', '#/conversations', '#/escalates-configuration', '3 awaiting reply'),
+      new view.ActivePackagesViewPartial('Open conversations', '#/conversations', '#', '20 open conversations'),
```

Comment:
this 20 is likely going to be different to the 20 below... let's change it to 30 so as not to confuse things

Reply:
OKay. Chaged

Reply:
OKay. Changed